joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -195,7 +195,27 @@ controller_interface::return_type JointTrajectoryController::update()
       }
       if (has_velocity_command_interface_)
       {
-        assign_interface_from_point(joint_command_interface_[1], state_desired.velocities);
+        if (!use_closed_loop_pid_adapter)
+        {
+          assign_interface_from_point(joint_command_interface_[1], state_desired.velocities);
+        }
+        else
+        {
+          std::vector<double> command(pids_.size());
```

Comment:
Please initialize this variable in the configure method and not on each update

Reply:
ros-controls/ros2_controllers@68d838d